guidelines/style-guidelines.md

```diff
@@ -0,0 +1,162 @@
+* names of people, places and things, including buildings and brands
+* titles, such as: of books, albums, recordings
+
+'Types' of places are not capitalized, unless referring to a specific place.
+For example: "I like the library" - and "I like my local Library"
+
+Generally, terms are not proper nouns, so should not be capitalised. Technical terms are not proper nouns. But if a word or term is branded as a distinct thing, treat it as a proper noun.
+
+#### Headings
+
+We use sentence case so only the first letter is upper case. We never link headings.
+
+#### Quotation marks
+
+We use double quotation marks for:
+* exact quotations
+* direct speech.
+
+We use single quotation marks for:
+* technical terms (the first time it is used)
+* classification descriptors
+* titles of documents or publications
+
+
+## Links
+
+We:
+* link to relevant content on our website before we link to external websites 
+* never link headings
+* write descriptive links that tell you what you’ll find when you follow them - not 'click here'
+* don't set links to open in a new tab or window, with exceptions to be decided case by case.
+
+
+## Lists
+
+We use bulleted lists (coded as unordered lists) to list items or points, and numbered lists (coded as ordered lists) for processes where the order of steps is important.
+
+We try to:
+* keep our lists short (2–7 items)
+* only use 1 level of nesting.
+
+We use 2 types of bulleted lists – single-sentence lists and multi-sentence lists.
+
+When we’re writing a single-sentence list, we:
+* start with a stem sentence that all the points have in common
+* start each point in lower case, and only use a full stop on the last point
+* don't use 'and' or 'or' on the second-to-last point
+* check that each point makes a full sentence when read with the stem.
+
+Multi-sentence lists are introduced by a complete sentence.
+* Each point in the list is also a complete sentence.
+* Each point can be 1–3 sentences long.
+* Each point begins with a capital letter and ends with a full stop.
+
+
+## Numbers
+
+In general:
+* we use numerals instead of words when we write numbers – this helps you scan our content
+* we use commas and no spaces to separate thousands when the number is over 10,000
+* when we’re talking about numbers in the millions, we use the word 'million' instead of writing out the number in full
+* we use spaces to separate groups of numbers when we write phone numbers.
+
+
+## Symbols, currency and abbreviations
+
+#### Symbols
+
+We use:
+* % – not 'percent' or 'per cent'
+* & – only if it’s part of a brand name
+* KB for kilobyte, MB for megabyte, and GB for gigabyte, for example 122 KB.
+
+#### Currency
+
+We put both the currency code and currency symbol before any amounts of money we write.
+We don't use spaces between the code, symbol and amount.
+
+For example:
+* 'If you’re a United States citizen, you pay USD$640'.
+* 'If you’re an Australian citizen it costs AUD$890'.
```

Comment:
```suggestion
```

Reply:
Keep in mind if you mark stuff as "resolved" then your comments get hidden :) (maybe it was intentional, but just in case)